HMCL/src/main/resources/assets/lang/I18N_ua.properties

```diff
@@ -0,0 +1,1322 @@
+account.methods.microsoft.error.unknown=Не вдалося увійти, код помилки: %d.
+account.methods.microsoft.error.wrong_verify_method=Будь ласка, увійдіть за допомогою свого пароля на сторінці входу в обліковий запис Microsoft і не використовуйте код підтвердження для входу.
+account.methods.microsoft.logging_in=Вхід...
+account.methods.microsoft.hint=Будь ласка, натисніть "Увійти" і скопіюйте код, показаний тут, щоб завершити процес входу у відкритому вікні браузера. \n\
+   Якщо токен, який використовується для входу в обліковий запис Microsoft, витік, ви можете натиснути "Скасувати авторизацію", щоб скасувати її.
+account.methods.microsoft.manual=Код вашого пристрою <b>%1$s</b>. Натисніть тут, щоб скопіювати. \n\
+   Після натискання "Увійти" ви повинні завершити процес входу у відкритому вікні браузера. Якщо воно не з'явиться, ви можете перейти до %2$s вручну. \n\
+   Якщо токен, який використовується для входу в обліковий запис Microsoft, витік, ви можете натиснути "Скасувати авторизацію", щоб скасувати її.
+account.methods.microsoft.makegameidsettings=Створити профіль / Редагувати ім'я профілю
+account.methods.microsoft.profile=Профіль облікового запису
+account.methods.microsoft.purchase=Купити Minecraft
+account.methods.microsoft.snapshot=Ви використовуєте неофіційну збірку HMCL. Завантажте офіційну збірку для входу.
+account.methods.microsoft.snapshot.website=Офіційний сайт
+account.methods.offline=Автономно
+account.methods.offline.name.special_characters=Використовуйте лише літери, цифри та підкреслення (макс. 16 символів)
+account.methods.offline.name.invalid=Рекомендується використовувати лише англійські літери, цифри та підкреслення для імені користувача, довжина не повинна перевищувати 16 символів. \n\
+   \n\  · Допустимі: HuangYu, huang_Yu, Huang_Yu_123; \n\
+   \n\  · Недопустимі: Huang Yu, Huang-Yu_%%%, Huang_Yu_hello_world_hello_world. \n\
+   Використання недопустимого імені користувача не дозволить вам приєднатися до більшості серверів і може конфліктувати з деякими модами, що призведе до збою гри.
+account.methods.offline.uuid=UUID
+account.methods.offline.uuid.hint=UUID - це унікальний ідентифікатор для гравців Minecraft, і кожен лаунчер може генерувати UUID по-різному. Зміна його на той, що генерується іншими лаунчерами, дозволяє вам зберегти предмети у вашому інвентарі автономного облікового запису. \n\
+   Ця опція призначена лише для досвідчених користувачів. Ми не рекомендуємо змінювати цю опцію, якщо ви не знаєте, що робите.
+account.methods.offline.uuid.malformed=Недійсний формат.
+account.methods.forgot_password=Забули пароль
+account.methods.ban_query=Запит блокування
+account.missing=Немає облікових записів
+account.missing.add=Натисніть тут, щоб додати.
+account.move_to_global=Перетворити на глобальний обліковий запис
+Інформація про обліковий запис буде збережена у файлі конфігурації поточного каталогу користувача системи.
```

Comment:
The format here is wrong. You should use `\n` to represent a newline.

If you think they are too long to fit on one line, end the first line with `\n\`. A backslash at the end of a line indicates that the value will be continued to the next line.

For more information about properties files, see its documentation (it's very brief, so I recommend you read it first): [Properties File Format](https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html)